app/src/main/res/values/strings.xml

```diff
@@ -328,4 +329,25 @@
     <string name="local_bundle_description">Import local files from your storage, does not automatically update</string>
     <string name="remote_bundle_description">Import remote files from a URL, can automatically update</string>
     <string name="recommended">Recommended</string>
+
+    <string name="installation_failed_dialog_title">Installation failed</string>
+    <string name="installation_cancelled_dialog_title">Installation cancelled</string>
+    <string name="installation_blocked_dialog_title">Installation blocked</string>
+    <string name="installation_conflict_dialog_title">Installation conflict</string>
+    <string name="installation_incompatible_dialog_title">Installation incompatible</string>
+    <string name="installation_invalid_dialog_title">Installation invalid</string>
+    <string name="installation_storage_issue_dialog_title">Installation storage issue</string>
```

Comment:
Does this just mean there's not enough storage? If so, how about `Not enough storage` or something similar, so the user can clearly tell what the problem is if they just read the title and not anything else.

Reply:
Unfortunately, there is no specification on what kind of issue could occur. Not enough storage is the most likelihood, though; other storage issues could occur, for example no file system permission for some reason, but due to the likelihood I'd keep it simple with "Not enough storage"